src/classes/Metrics.js

```diff
@@ -510,10 +510,16 @@ class Metrics {
 	 */
 	_getThemeValue() {
 		const { current_theme } = conf;
-		if (current_theme === 'midnight-theme') {
-			return 1;
+		switch (current_theme) {
+			case 'midnight-theme':
```

Comment:
Nothing to change here & now, but just a note that we should remember to update this bit to `dark-blue-theme` in the final theme framework PR

Reply:
I spoke with Frank a while ago and I think we're only visually updating the theme to dark-blue-theme and not the theme variables themselves so we can prevent breakage for users on older versions of the extension that request a midnight-theme from the account project which would no longer exist. We do however have a ticket in the backlog to do this properly